client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceTool.js

```diff
@@ -0,0 +1,434 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React, { PureComponent } from 'react';
+
+import PlayIcon from 'icons/Play.svg';
+
+import CamundaAPI from '../shared/CamundaAPI';
+import KeyboardInteractionTrap from '../shared/KeyboardInteractionTrap';
+
+import StartInstanceConfigModal from './StartInstanceConfigModal';
+
+import css from './StartInstanceTool.less';
+
+import { Fill } from '../../../app/slot-fill';
+
+import {
+  DropdownButton,
+  Icon
+} from '../../../app/primitives';
+
+const START_DETAILS_CONFIG_KEY = 'start-instance-tool';
+
+const START_INSTANCE_FAILED = 'Starting process instance failed';
+
+const PROCESS_DEFINITION_CONFIG_KEY = 'process-definition';
+
+export default class StartInstanceTool extends PureComponent {
+
+  state = {
+    modalState: null,
+    activeTab: null
+  }
+
+  componentDidMount() {
+
+    const {
+      subscribe
+    } = this.props;
+
+    subscribe('app.activeTabChanged', ({ activeTab }) => {
+      this.setState({ activeTab });
+    });
+
+    this.START_ACTIONS = [
```

Comment:
Is there any particular reason to set this on mount?

Reply:
Nope. I updated it.